.github/ISSUE_TEMPLATE/qa_issue_template.md

```diff
@@ -0,0 +1,32 @@
+---
+name: "Testing type, Checklist # "
```

Comment:
Tbh, I'm not sure I understand this structure ... could you please share a link what this metadata affects?